bin/appbundle-updater

```diff
@@ -175,42 +196,42 @@ CHEFDK_APPS = [
     "chef-vault",
     "chef/chef-vault",
     "development",
-    "#{bin_dir.join("rake")} install"
+    "#{vendor_bin_dir.join("rake")} install"
   ),
   App.new(
     "cookstyle",
     "chef/cookstyle",
     "development debug docs",
-    "#{bin_dir.join("rake")} install"
+    "#{vendor_bin_dir.join("rake")} install"
   ),
   App.new(
```

Comment:
We can remove `foodcritic`

Reply:
removed in https://github.com/chef/appbundle-updater/pull/107/commits/6ee4d2613439292b58c64673a96a6302d90eac28